packaging/installer/methods/macos.md

```diff
@@ -5,6 +5,10 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/instal
 
 # Install Netdata on macOS
 
+**Disclaimer:** Currently the Netdata Agent can't connect to the Netdata Cloud when used in MacOS. To experience the full feature-set of Netdata:
+1) Download [Docker Desktop for Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac)
+2) Follow the instructions for [installing Netdata using Docker](../docker/../../../README.md).
```

Comment:
In that case, you effectively monitor Linux virtual machine, not the host itself.

Reply:
We could say something like for testing purposes and for real production value you need linux machine. Either way, we should say that currently it's broken or remove it alltogether.

Reply:
The key here is that Netdata Cloud does not work. As of the release of 1.31.0 though (planned for tomorrow), we should otherwise be as feature complete as reaslistically possible on macOS  compared to Linux.

And even then, it’s only installs with Homebrew where it does not work, installs with our kickstart script do (AFAIK) have working cloud support.

I think the correct way to handle this is to say that the Homebrew package does not currently have working Cloud support, but we are working on fixing it and anyone who wants to just test out Netdata Cloud can either use our Docker images with Docker Desktop for Mac, or install using our kickstart installer.

Reply:
I concur with @Ferroin. If kickstart works and provides Cloud support, that should 100% be the recommendations, not Docker. We should, actually, remove all the other options if kickstart does indeed work at this point.

Reply:
Kickstart still has some significant UX issues, most notably that it currently requires you to explicitly specify an install prefix.

Reply:
Just a note, and we should check more, I'm not sure if kickstart works ok on M1 macs...

Reply:
If our installation _shell script_ does not work correctly on Apple M1 hardware, then I would argue that that is a bug in macOS, not our code.

If it’s something in the build system though, then that affects the Homebrew formula as well, not just the kickstart installer.

Reply:
Last time I checked, the kickstart script failed to build for netdata cloud.

Reply:
Drive-by comment: if documentation contains disclaimers about things that are not working, they should also have a github issue link to let users get updates about the latest status.

Reply:
This is a good comment @vkalintiris